iotdb/src/main/java/org/apache/iotdb/db/postback/conf/PostBackSenderDescriptor.java

```diff
@@ -87,46 +83,42 @@ private void loadProps() {
     try {
       properties.load(inputStream);
 
-      conf.serverIp = properties.getProperty("server_ip", conf.serverIp);
-      conf.serverPort = Integer
-          .parseInt(properties.getProperty("server_port", conf.serverPort + ""));
+      conf.setServerIp(properties.getProperty("server_ip", conf.getServerIp()));
+      conf.setServerPort(Integer
+          .parseInt(properties.getProperty("server_port", conf.getServerPort() + "")));
```

Comment:
int+"" is inefficient comparing with String.valueOf()

Reply:
fixed